scripts/build-consumer.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/sh
```

Comment:
Will this script work for checkout, too? If not, can we make it clear it only works with the admin?

Reply:
Yes, it does work with Checkout  - you can test it out :)